### ISSUE TITLE
Data model stored in Context Broker is wrong

### ISSUE THREAD

**Issue by jmcanterafonseca** [Bitergia/fiware-devguide-app#44]
Once you update to Orion >= 0.28 data model needs to be fixed: Here is the data model in normalized mode:

When creating data, if you issue a POST to /v2/entities with the JSON described below it will create the entities correctly. 

``` js
{
    "address": {
        "type": "PostalAddress",
        "value": {
            "addressLocality": "Alegr\u00eda-Dulantzi",
            "addressRegion": "Araba",
            "postalCode": "01240",
            "streetAddress": "Cuesta de las Cabras Aldapa 2"
        }
    },
    "aggregateRating": {
        "value": {
            "ratingValue": 3,
            "reviewCount": 1
        }
    },
    "capacity": {
        "value": 120
    },
    "department": {
        "value": "Franchise3"
    },
    "description": {
        "value": "Restaurante de estilo sidrer\u00eda ubicado en Alegria-Dulantzi. Adem\u00e1s de su men\u00fa del d\u00eda y carta, tambi\u00e9n ofrece men\u00fa de sidrer\u00eda. El men\u00fa del d\u00eda cuesta 9 euros. Los fines de semana la especialidad de la casa son las alubias con sacramentos. En lo que a bebidas se refiere, hay una amplia selecci\u00f3n adem\u00e1s de la sidra. Cabe destacar que se puede hacer txotx. La capacidad del establecimiento es de 50 personas pero la sidrer\u00eda no dispone de aparcamiento.%5cn%5cnHORARIO: %5cn%5cnLunes a domingo: 9:00-17:00 y 19:00-23:00."
    },
    "id": "Elizalde",
    "occupancyLevels": {
        "metadata": {
             "timestamp": {
                "value": 1454350000000.0
            }
        },
        "value": 0
    },
   "location": {
        "type": "geo:point",
        "value": "42.8404625, -2.5123277"
    },
    "priceRange": {
        "value": 0
    },
    "telephone": {
        "value": "945 400 868"
    },
    "type": "Restaurant"
}

```

**Comment by albertinisg** [comment]
I've also modified the models regarding reviews and reservation. After continuing, I would like to double check that the modifications I've made are correct. 
I have doubts with the fields containing dates, the [Orion documentation](https://jsapi.apiary.io/previews/null/introduction/specification/special-attribute-types) now uses ISO8601 formats, but I couldn't find how to perform the queries we are performing now using the unix timestamp format (i.e, filter between dates). Also, I'm wondering if those date fields should be added as metadata or not.

For reservations:

``` js
{  
   "type":"FoodEstablishmentReservation",
   "id":"Elizalde-NyH5u_QTg",
   "reservationStatus":{  
      "value":"Confirmed"
   },
   "underName":{  
      "type":"Person",
      "value":"user7"
   },
   "reservationFor":{  
      "type":"FoodEstablishment",
      "value":"Elizalde"
   },
   "address":{  
      "type":"postalAddress",
      "value":{  
         "streetAddress":"Cuesta de las Cabras Aldapa 2",
         "addressRegion":"Araba",
         "addressLocality":"Alegría-Dulantzi",
         "postalCode":"01240"
      },
   },
   "startTime":{  
      "value":1448009647286
   },
   "partySize":{  
      "value":9
   }
}
```

For reviews:

``` js
{  
   "type":"Review",
   "id":"Elizalde-41eYKuXag",
   "itemReviewed":{  
      "type":"Restaurant",
      "value":"Elizalde"
   },
   "reviewRating":{  
      "type":"Rating",
      "value":2
   },
   "author":{  
      "type":"Person",
      "value":"user10"
   },
   "reviewBody":{  
      "value":"Body review"
   },
   "dateCreated":{  
      "value":1458211423349
   },
   "publisher":{  
      "type":"Organization",
      "value":"Bitergia"
   }
}
```

**Comment by jmcanterafonseca** [comment]
several comments, 

dates should be captured as 

``` json
{
  "value": "date_in_iso_8601_format",
  "type": "date"
}
```

latest version of Orion supports date filtering in queries, please check

type of address must be 'PostalAddress' and not 'postalAddress'

**Comment by albertinisg** [comment]
Fixed in #69 (that inherits from #68)
